website_menu_icon/models/website_menu.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Radovan Skolnik <radovan@skolnik.info>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class Menu(models.Model):
+
+    _inherit = "website.menu"
+
+    image = fields.Char("Icon", required=False, translate=True)
+    image_type = fields.Selection(
```

Comment:
This can easily be confused with a raw binary `image` field, so better rename it. Also... translatable? :thinking: 

```suggestion
    icon = fields.Char()
    icon_type = fields.Selection(
```